spec/utils_spec.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-env mocha */
-/* global expect */
-describe('Utils', function () {
-  var Promise = window.Promise || require('native-promise-only')
```

Comment:
Do we still need to import Promise here?

Reply:
I did a bit of digging, but that line is very old from when we were running in phantom, not in chrome. We only run in chrome now, so Promise will always be available.